box-setup/install-agent-clis.sh

```
@@ -0,0 +1,270 @@
+
+log "codex"
+npm_install codex "@openai/codex"
+
+# ----------------------------------------------------------------- claude ----
+log "claude code"
+if have claude; then
+  note "claude already present ($(claude --version 2>&1 | head -1))"
+  mark claude "present"
+else
+  curl -fsSL https://claude.ai/install.sh | bash >/dev/null 2>&1
+  have claude && mark claude "installed" || mark claude "FAILED (claude.ai/install.sh)"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)